evergreen.yml

```diff
@@ -33,51 +65,144 @@ functions:
         shell: bash
         working_dir: mongo-powerbi-connector
         script: |
+          export MONGO_POWERBI_CONNECTOR_VERSION=$(git describe --always | sed 's/^v//g')
+          export UPDATE_PLUGIN_VERSION=true
+          git describe --exact-match HEAD || UPDATE_PLUGIN_VERSION=false
+
           cat <<EOT > expansions.yml
+          MONGO_POWERBI_CONNECTOR_VERSION: "$MONGO_POWERBI_CONNECTOR_VERSION"
+          UPDATE_PLUGIN_VERSION: "$UPDATE_PLUGIN_VERSION"
           prepare_shell: |
             set -o errexit
             export PATH="$PATH"
+            export MONGO_POWERBI_CONNECTOR_VERSION="$MONGO_POWERBI_CONNECTOR_VERSION"
+            export UPDATE_PLUGIN_VERSION="$UPDATE_PLUGIN_VERSION"
           EOT
     - command: expansions.update
       params:
         file: mongo-powerbi-connector/expansions.yml
 
-  "build connector":
+  "install power query tools":
     - command: shell.exec
       params:
         shell: bash
         working_dir: mongo-powerbi-connector
         script: |
-          echo 'PLACEHOLDER' > mongodb-powerbi.pqx
+          ${prepare_shell}
+          curl  -L -o tools.zip https://www.nuget.org/api/v2/package/Microsoft.PowerQuery.SdkTools/2.111.5
```

Comment:
Question: Does the SDK version change often? What are we planning to do? Can we stay on an older version of the SDK for as long as we want or does PowerBI require your connector to use a specific version to let you run it?

Reply:
They seem to be updating the versions often, about 5 versions in the last two months [here](https://www.nuget.org/packages/Microsoft.PowerQuery.SdkTools#versions-body-tab).  I tested without specifying the `2.111.5` version and it was able to download the package, I'll make that change.